zwave_js_server/model/node.py

```diff
@@ -216,6 +220,21 @@ def receive_event(self, event: Event) -> None:
 
         self.emit(event.type, event.data)
 
+    async def set_value(self, val: Union[Value, str], new_value: Any) -> None:
```

Comment:
I'd like to clarify what kind of value the first parameter is. Not sure what's best.
```suggestion
    async def set_value(self, value_id: Union[Value, str], new_value: Any) -> None:
```

Reply:
value_id clashes with the method value_id

Reply:
maybe just value ?

Reply:
Or we decide to use `value` to mean specifically `Value` instances, but it's hard to keep that consistent I'm fearing.

Reply:
Well, I think the better option would be to use the naming of upstream lib,
so Value will become ZWaveValue and Node will become ZWaveNode etc.
So for variable names you can refer to zwave_value instead of just value

Reply:
Sounds good!

Reply:
Or go with your proposal. Use NodeValue class name and node_value as instance name

Reply:
We can do that in a follow up. I'm happy with this PR as is.

Reply:
Maybe write it on the todo list ?

Reply:
Can you make an issue?